src/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -13,6 +13,7 @@
 from turicreate.toolkits._model import Model as _Model
 import turicreate.toolkits._internal_utils as _tkutl
 from turicreate.toolkits._main import ToolkitError as _ToolkitError
+from turicreate._deps.minimal_package import minimal_package_import_check
```

Comment:
You should hid this import, i.e. change it to:
`from turicreate._deps.minimal_package import minimal_package_import_check as _minimal_package_import_check`

General rule: if the file you're in is not hidden (i.e. the file name doesn't start with an underscore), then you need to hide anything the user should never use.

Reply:
sounds reasonable. Will change it when I have time.